boards/arm/s32z270dc2_r52/s32z270dc2_rtu0_r52.dts

```diff
@@ -5,7 +5,6 @@
  */
 
 /dts-v1/;
-#include <dt-bindings/clock/nxp_s32z2_clock.h>
```

Comment:
pls apply the same change to https://github.com/zephyrproject-rtos/zephyr/blob/2297b0f7c32d94af34bab109dafdf61b5fe4743a/boards/arm/s32z270dc2_r52/s32z270dc2_rtu1_r52.dts#L8

Reply:
done